Gems/Atom/RPI/Code/Source/RPI.Public/DynamicDraw/DynamicDrawContext.cpp

```diff
@@ -493,6 +493,7 @@ namespace AZ
 
             DrawItemInfo drawItemInfo;
             RHI::DrawItem& drawItem = drawItemInfo.m_drawItem;
+            drawItem.m_enabled = true;
```

Comment:
Isnt m_enabled true by default?

Reply:
Updated! You're right, the original intent was to enable drawing by default.